tutorials/source-modeling/plot_background_freesurfer_mne.py

```diff
@@ -331,7 +331,7 @@ def imshow_mri(data, img, vox, xyz, suptitle):
 # sphere, a given vertex in the source (sample) mesh can be mapped easily
 # to the same location in the destination (fsaverage) mesh, and vice-versa.
 
-renderer_kwargs = dict(bgcolor='w', smooth_shading=False)
+renderer_kwargs = dict(bgcolor='w')
```

Comment:
This is the one place we actually don't want smooth shading. Is there any way to disable it on this example? It makes it clear that the mesh is decimated quite a bit. Or maybe we should just use wireframes to make the point? WDYT?

Reply:
Yeah I think wireframes could look good for the last plots:

https://mne.tools/dev/auto_tutorials/source-modeling/plot_background_freesurfer_mne.html#surface-decimation

Reply:
Using could be a bit messy. I will let this option then :+1: